src/Jumbotron.js

```diff
@@ -2,11 +2,21 @@ import React from 'react';
 import classNames from 'classnames';
 
 const Jumbotron = React.createClass({
+  propTypes: {
+    componentClass: React.PropTypes.node.isRequired
```

Comment:
You are correct.

What `PropType` would you suggest to use? React itself [uses `any`](https://github.com/facebook/react/blob/ba81b60ad8e93b747be42a03b797065932c49c96/src/addons/transitions/ReactTransitionGroup.js#L25).

What about the [other occurrences](https://github.com/react-bootstrap/react-bootstrap/search?utf8=%E2%9C%93&q=componentClass) of `componentClass?